eduvpn/exceptions.py

```diff
@@ -4,11 +4,11 @@
 # SPDX-License-Identifier: GPL-3.0+
 
 
-class EduvpnException(Exception):
+class EduvpnException(Exception):  # type: (Exception) -> None
```

Comment:
Dit is niet een functie die een Exceptie accepteert, maar een classe die overerft van een classe. De type doc string is dus verkeerd. Je kan ze gewoon weglaten hier, want er zijn geen methoden of functies. Het type van een class of object is de class, dus een type anotation is niet nodig.